receiver/hostmetricsreceiver/internal/scraper/cpuscraper/internal/cpu_utilization_calculator_test.go

```diff
@@ -0,0 +1,288 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/shirou/gopsutil/v3/cpu"
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/model/pdata"
+)
+
+func TestCpuUtilizationCalculator_Calculate(t *testing.T) {
```

Comment:
Is this test thread-safe? If so we should call `t.Parallel()`
```suggestion
func TestCpuUtilizationCalculator_Calculate(t *testing.T) {
	t.Parallel()
```